packages/flame/lib/src/extensions/image.dart

```diff
@@ -90,4 +90,26 @@ extension ImageExtension on Image {
     }
     return fromPixels(newPixelData, width, height);
   }
+
+  /// Resizes this image to the given [newSize].
```

Comment:
probably worth adding a note here that this is one of those kinds of _expensive_ operations. ie not to be done on gameloops